### PR TITLE
Added files

### PR DESCRIPTION
**Pull request by NishanthSubramanian**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        </div>
+<!--
+        <div class="home-page-banner">
+            <div width=100% height=100% style=" position: absolute">
+                <div class="card-panel teal lighten-2">
+                    <h1 style="color: #FFF">Hello, I'm Nishanth Subramanian</h1>
+                </div>
+            </div>
+        </div>
+-->
```
Comment (salman-bhai) [inline index.html]: Remove this commented code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            </div>
+          </div>
+          <div class="footer-copyright">
+          </div>
+        </footer>
+        <!--Import jQuery before materialize.js-->
+
+      <script type="text/javascript" src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/js/materialize.min.js"></script>
+    <script src="home.js"></script>
```
Comment (salman-bhai) [inline index.html]: Where is this file?

File: projects.html [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+      <title>My Projects</title>
+    </head>
+
+    <body>
+      <div class="navbar-fixed">
+       <nav>
+        <div class="nav-wrapper" style="position: relative " >
+            <a href="index.html" class="brand-logo right">NS</a>
+            <ul id="nav-mobile" class="hide-on-med-and-down">
+                <li><a href="index.html">Home</a></li>
```
Comment (salman-bhai) [inline projects.html]: Remove `index.html` and change it to */*

File: projects.html [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+        <div class="card-action">
+          <a href="#">This is a link</a>
+        </div>
+      </div>
+    </div>
+  </div>
+          </div>
+                 <br>
+            <div class="container">
+  <div class="col s12 m7">
```
Comment (salman-bhai) [inline projects.html]: Why m7 ?

File: projects.html [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+        <div class="card-action">
+          <a href="#">This is a link</a>
+        </div>
+      </div>
+    </div>
+  </div>
+          </div>
+                 <br>
+         <div class="container">
+  <div class="col s12 m7">
```
Comment (salman-bhai) [inline projects.html]: Why m7?

File: projects.html [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+      </div>
+    </div>
+  </div>
+          </div>
+          <br><br><br>
+           
+                   <footer class="page-footer teal lighten-2">
+          <div class="container" >
+            <div class="row">
+              <div class="col l6 s12">
```
Comment (salman-bhai) [inline projects.html]: I think this should be s6

Comment (salman-bhai) [reply projects.html]: Oh wait ignore this.

File: projects.html [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+              </div>
+            </div>
+          </div>
+          <div class="footer-copyright">
+          </div>
+        </footer>
+<!--
+          <div class="footer-copyright">
+          </div>
+-->
```
Comment (salman-bhai) [inline projects.html]: Remove this if it is not needed.

File: skill.html [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+    <head>
+      <!--Import Google Icon Font-->
+      <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+      <!--Import materialize.css-->
+       <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css">
+        <link rel="stylesheet" href="css/style.css">
+      <!--Let browser know website is optimized for mobile-->
+      <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+
+      <title>Nishanth Subramanian</title>
```
Comment (salman-bhai) [inline skill.html]: Change the title according to the page. No point having the same title everywhere.

File: skill.html [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+        </div>
+<!--
+        <div class="home-page-banner">
+            <div width=100% height=100% style=" position: absolute">
+                <div class="card-panel teal lighten-2">
+                    <h1 style="color: #FFF">Hello, I'm Nishanth Subramanian</h1>
+                </div>
+            </div>
+        </div>
+-->
```
Comment (salman-bhai) [inline skill.html]: Remove this if it is not needed.

File: projects.html (hunk 5 above)
Comment (NishanthSubramanian) [reply projects.html]: medium-7

File: projects.html (hunk 3 above)
Comment (NishanthSubramanian) [reply projects.html]: you mean \ 
done.

**Pull request closed without merging**